README.md

```diff
@@ -125,16 +125,48 @@ The [`test/setup-ca-kubernetes.sh`](test/setup-ca-kubernetes.sh) script shows ho
 
 A production deployment can improve upon that by using some other key delivery mechanism, like for example [Vault](https://www.vaultproject.io/).
 
-### Dynamic provisioning
-
-The following diagram illustrates how the PMEM-CSI driver performs dynamic volume provisioning in Kubernetes:
-![sequence diagram](/docs/images/sequence/pmem-csi-sequence-diagram.png)
 <!-- FILL TEMPLATE:
 * Target users and use cases
 * Design decisions & tradeoffs that were made
 * What is in scope and outside of scope
 -->
 
+### Volume Persistency
+
+Normally, volumes are provided by a storage backend that is independent of a particular node. When a node goes offline, the volume can be mounted elsewhere. But PMEM volumes are *local* to node and thus can only be used on the node where they were created. This limits the applications using PMEM volume cannot freely move between nodes. This limitation needs to be considered when designing and deploying applications that are to use *local storage*.
```

Comment:
can we express the point more precisely? Just "Normally" does not describe a definitive state. What about saying "In a typical CSI deployment" (or something like that)

Reply:
Agreed, Done